.github/workflows/metrics.yaml

```diff
@@ -0,0 +1,21 @@
+name: Scheduled Metrics
+on:
+  schedule:
+  # * is a special character in YAML so you have to quote this string
+  - cron:  '0 0 * * *'
+
+jobs:
+  metrics:
+    name: Collect and Send Metrics
+    runs-on: ubuntu-20.04
+    env:
+      SEGMENT_WRITE_KEY: ${{ secrets.SEGMENT_WRITE_KEY }}
```

Comment:
what is the ownership of this key? Can it expire?

Reply:
Devtools Telemetry owns , but the workspace is per-team basis. No expiration for this key.

See https://source.redhat.com/groups/public/appservices/wiki/helm_metrics_collection_with_segment_and_woopra